{"name": "testLogin", "status": "failed", "statusDetails": {"message": "AssertionError: assert False", "trace": "self = <testCases.test_validRegister.Test_Valid_Registration object at 0x00000201BCCFF048>\n\n    def testLogin(self):\n        self.driver = webdriver.Firefox(executable_path=\"Drivers\\\\geckodriver.exe\")\n        self.driver.get(ReadConfig.getLoginPageURL())\n        loginp = LoginPage(self.driver)\n        loginp.setUserName(self.email)\n        loginp.setPassword(self.password)\n        loginp.clickLogin()\n        time.sleep(5)\n        currentUrl = self.driver.current_url\n        if currentUrl == ReadConfig.getHomePageURL():\n            self.driver.close()\n            assert True\n        else:\n            allure.attach(self.driver.get_screenshot_as_png(),\n                          name=\"Failed Login After Registration\",\n                          attachment_type=AttachmentType.PNG)\n            self.driver.close()\n>           assert False\nE           assert False\n\ntestCases\\test_validRegister.py:60: AssertionError"}, "attachments": [{"name": "Failed Login After Registration", "source": "da1e3b25-4d84-4e93-b2e8-200544243bf8-attachment.png", "type": "image/png"}], "start": 1602684018254, "stop": 1602684079506, "uuid": "73e8ca8d-f793-4cf1-9a38-7365e249b9ee", "historyId": "9ab8ed280ab7b9f228928a303544cd2f", "testCaseId": "d54c8376626ddc0bdcf29b9e5ca0f3c3", "fullName": "testCases.test_validRegister.Test_Valid_Registration#testLogin", "labels": [{"name": "parentSuite", "value": "testCases"}, {"name": "suite", "value": "test_validRegister"}, {"name": "subSuite", "value": "Test_Valid_Registration"}, {"name": "host", "value": "DESKTOP-3NDT0U6"}, {"name": "thread", "value": "11248-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testCases.test_validRegister"}]}